Ginger/GingerCoreNET/ActionsLib/UI/Web/ActAccessibilityTesting.cs

```diff
@@ -0,0 +1,203 @@
+                }
+                return mPlatforms;
+            }
+        }
+
+        public override string ActionEditPage { get { return "ActAccessibilityTestingEditPage"; } }
+
+        public override void ActionUserRecommendedUseCase(ITextBoxFormatter TBH)
+        {
+            TBH.AddText("Action to Test Accessibility Testing on Web");
+            TBH.AddLineBreak();
+            TBH.AddText("Accessibility testing action is using Axe Core tool");
+            TBH.AddLineBreak();
+            TBH.AddText("Accessibility testing is the practice of making your web and mobile apps usable to as many people as possible");
+        }
+
+        public PlatformAction GetAsPlatformAction()
+        {
+            PlatformAction platformAction = new PlatformAction(this);
+
+
+
+
+            foreach (ActInputValue aiv in this.InputValues)
+            {
+                if (!platformAction.InputParams.ContainsKey(aiv.Param))
+                {
+                    platformAction.InputParams.Add(aiv.Param, aiv.ValueForDriver);
+                }
+            }
+
+
+            return platformAction;
+        }
+
+        public string GetName()
+        {
+            return "AccessibilityAction";
+        }
+
+        public new static partial class Fields
+        {
+            public static string Target = "Target";
+            public static string ValueUC = "ValueUC";
+            public static string ElementLocateValue = "ElementLocateValue";
+            public static string ElementLocateBy = "ElementLocateBy";
+            public static string PomGUID = "PomGUID";
+            public static string Standard = "Standard";
+            public static string ElementType = "ElementType";
+            public static string ValueToSelect = "ValueToSelect";
+        }
+
+        public enum eTarget
+        {
+            [EnumValueDescription("Page")]
+            Page,
+            [EnumValueDescription("Element")]
+            Element,
+        }
+
+        public override eLocateBy LocateBy
+        {
+            get
+            {
+                return GetOrCreateInputParam<eLocateBy>(Act.Fields.LocateBy, eLocateBy.NA);
+            }
+            set
+            {
+                AddOrUpdateInputParamValue(Act.Fields.LocateBy, value.ToString());
+                OnPropertyChanged(Act.Fields.LocateBy);
+                OnPropertyChanged(Act.Fields.Details);
+            }
+        }
+
+        public override string LocateValue
+        {
+            get
+            {
+                return GetOrCreateInputParam(Act.Fields.LocateValue).Value;
+            }
+            set
+            {
+                AddOrUpdateInputParamValue(Act.Fields.LocateValue, value);
+                OnPropertyChanged(Act.Fields.LocateValue);
+                OnPropertyChanged(Act.Fields.Details);
+            }
+        }
+
+        public enum eTags
+        {
+            [EnumValueDescription("wcag2a")]
```

Comment:
add familiar standard names in the enum description

Reply:
Added